### DISCUSSION TITLE
keep live monitoring of security onion active

### DISCUSSION THREAD

**Discussion by pavlos1982** [Security-Onion-Solutions/securityonion#5486] [General] [answered]
Hi, after a while, I noticed that the security onion page goes inactive and time out, so I have to re-login the portal. Is there a way to keep the live monitoring active as I plan to put the live alert monitoring to a dedicated open screen? See the screenshot below. After a while of monitoring and leaving the refresh interval for 5 sec, it seems to time out or fails where I have to re-login to reaccess the alerts?

![monitoringso](https://user-images.githubusercontent.com/5363713/133074488-ed01a52a-472b-46c9-a23b-cb3ec3dbf466.JPG)

**Comment by jertel** [comment]
How long is "a while"? 

See #4575.

**Reply by pavlos1982** [reply]
about 30 mins then I get this error status and I have to re-logged in. I will check the document thanks

**Comment by dougburks** [comment]
Also see https://github.com/Security-Onion-Solutions/securityonion/issues/4586. I've added these steps to the SOC documentation:
https://docs.securityonion.net/en/2.3/soc.html

**Reply by pavlos1982** [reply]
thanks Doug, I will have a look

**Reply by pavlos1982** [reply]
it seems this is for the evaluation version but I have the standalone version and when trying to access the path :**/opt/so/saltstack/local/pillar/minions/** I get access denied, and is there a way to set it indefinitely until the user sign out?

**Reply by dougburks** [reply]
You need root or sudo privileges to access `/opt/so/saltstack/local/pillar/minions/ `.

**Comment by pavlos1982** [comment]
I did try with sudo getting this error:
![errorsudo](https://user-images.githubusercontent.com/5363713/134124564-5b67ac5a-4736-406d-8c7d-72b740e1be66.JPG)

**Reply by dougburks** [reply]
Try becoming root:
```
sudo -i
```

**Reply by pavlos1982** [reply]
I have amended as recommend see screen shot:


the error result I am getting despite running sudo so-soc-restart after the change:

![errorsec1](https://user-images.githubusercontent.com/5363713/134665387-0435bd8a-158a-4c25-859d-2872c9223c57.JPG)

**Reply by pavlos1982** [reply]
it seems to still time out and I have to re logged in

**Reply by dougburks** [reply]
First, please note that you may want to redact keys like `kratoskey` when posting to a public forum like this.

Looking at your screenshot, it appears that `sessiontimeout` is not indented as shown at https://docs.securityonion.net/en/2.3/soc.html. Please try indenting and then restarting SOC to see if that changes anything. You can also check the logs at /opt/so/log/soc/ for additional clues.

**Reply by pavlos1982** [reply]
Thanks I made the change recommended see screenshot and I will report back:
![timeout1](https://user-images.githubusercontent.com/5363713/136016448-c1e96df3-9afb-4ec0-b72b-9095afd17bf2.JPG)

**Reply by pavlos1982** [reply]
after 30 mins same message again:
![timeout33](https://user-images.githubusercontent.com/5363713/136021022-68c55a0c-6871-425c-a699-bb196915027f.JPG)

**Reply by dougburks** [reply]
I wouldn't think you would get a timeout after only 30 minutes. Is your system healthy or could it be overloaded? What is the load average? Are you able to duplicate this on a fresh installation (perhaps just in a VM)?

**Comment by jertel** [comment, marked as answer]
Since you're the only one reporting this issue my best guess is you have something in your network, like a proxy or related device, that's interfering. I suggest trying some alternatives:
- Login using another browser
- Login using another computer
- Install SO on a completely separate network and compare results

**Reply by pavlos1982** [reply]
Thanks it is sorted now
